privacyidea/lib/eventhandler/usernotification.py

```diff
@@ -140,7 +140,31 @@ def actions(cls):
                 "reply_to": {
                     "type": "str",
                     "required": False,
-                    "description": _("The Reply-To header in the sent email.")},
+                    "description": _("The Reply-To header in the sent email."),
```

Comment:
We have a problem here with old installations.

Installations before version 3.6 will have the dedicated email address in the `reply_to` database field.

~~~~
mysql> select * from eventhandleroption where eventhandleroption.Key = "reply_to";
+----+-----------------+----------+-----------------------+------+-------------+
| id | eventhandler_id | Key      | Value                 | Type | Description |
+----+-----------------+----------+-----------------------+------+-------------+
| 10 |               4 | reply_to | user@example.com |      |             |
+----+-----------------+----------+-----------------------+------+-------------+
1 row in set (0,00 sec)
~~~~

(See my comment below).

But we need to take care of this, that old configurations still work with version 3.6.
So basically there are two possibilities:

# A) Migrate the data

You need to migrate the evenhandleroption 

    reply_to -> reply_toEmail
    set reply_to = "Email"

# B) Name the fields this way, that it is backward compatible

* Introduce a new field "reply_to_type"
* Use "reply_to" for destinct email addresses
* And instead of having the field `"reply_to " + NOTIFY_TYPE.EMAIL` use normal `reply_to`, since this is the old field, that contains the email address.